### PR TITLE
Update wiki links in locales

### LINKED ISSUE #3724: Error in link referring to website
### Describe the bug

Error in link referring to website. This is the page that allows you to configure  VTX Tables 
 https://betaflight.com/docs/wiki/archive/VTX-tables

### To Reproduce

Video transmiter clic Go to this Page

### Expected behavior

change website link  https://betaflight.com/docs/wiki/guides/current/VTX-Tables

### Support ID

```txt
67890
```

### Flight controller

MatekF722SE

### Other components

_No response_

### How are the different components wired up (including port information)

_No response_

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #3724

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
